src/tests/Logary.Targets.Stackdriver.Tests/Program.fs

```diff
@@ -56,27 +56,19 @@ let target =
         |> Stackdriver.Impl.write
 
       Expect.equal subject.Severity LogSeverity.Warning "severity should be warning"
-      Expect.equal (subject.JsonPayload.Fields.["context"]) (Google.Protobuf.WellKnownTypes.Value.ForStruct(Struct.Parser.ParseJson("""{ "service" : "tests" }"""))) "should have correct context"
+      Expect.equal (subject.Labels.["service"]) "tests" "should have correct context"
       
     testCase "send" <| fun _ ->
+      let conf = Stackdriver.create targConf "test"
```

Comment:
I get repeated failures when I try to make this a proper test, by querying the count of messages posted, so I've had to neuter it.